lib/Doctrine/Annotations/Metadata/InternalAnnotations.php

```diff
@@ -58,12 +58,11 @@ public static function createMetadata() : MetadataCollection
                 IgnoreAnnotation::class,
                 new AnnotationTarget(AnnotationTarget::TARGET_ALL),
                 true,
-                [
-                    new PropertyMetadata(
-                        'names',
-                        new ListType(new StringType()),
-                        true
-                    ),
+                [new PropertyMetadata(
```

Comment:
This CS change looks fishy, original version is correct. Was this autofix?

Reply:
yes, good catch.

autofixing does strange things, I had `,,` instead of `,` after the last array element in a few places